### DISCUSSION TITLE
[Bug]custom hostname does not CNAME to this zone

### DISCUSSION THREAD

**Discussion by nguyenxly** [webstudio-is/webstudio-community#133] [Q&A]
Hi
I have successfully configured the pointer to the domain. But still error message
![z5301551823286_97586b3f0a2e671482e76a4d98efc045](https://github.com/webstudio-is/webstudio-community/assets/34223548/72bc928e-42f0-46f5-8f1f-b865f1c974c7)

**Comment by hidech** [comment]
In my case, I had to add a dot (.) to the end of my DNS provider's FQDN, which is shown the first line's "VALUE".
